testing/py-m2crypto/APKBUILD

```diff
@@ -2,30 +2,58 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-m2crypto
 _pkgname=M2Crypto
-pkgver=0.23.0
-pkgrel=1
+pkgver=0.25.1
+pkgrel=0
 pkgdesc="A crypto and SSL toolkit for Python"
-url="http://chandlerproject.org/bin/view/Projects/MeTooCrypto"
+url="https://pypi.python.org/pypi/M2Crypto/"
 arch="all"
 license="BSD"
-depends=""
-depends_dev=
-makedepends="python2-dev swig libressl-dev py-setuptools"
-install=""
-subpackages=""
+makedepends="swig libressl-dev python2-dev python3-dev py-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
 source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
 
-_builddir="$srcdir"/$_pkgname-$pkgver
+builddir="$srcdir"/$_pkgname-$pkgver
+prepare() {
+	local i
+	cd "$builddir"
+	for i in $source; do
+		case $i in
+		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+		esac
+	done
+}
```

Comment:
Please remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.